pkg/apis/serving/v1alpha1/revision_lifecycle_test.go

```diff
@@ -755,3 +764,80 @@ func TestPropagateAutoscalerStatus(t *testing.T) {
 	apitest.CheckConditionSucceeded(r.duck(), RevisionConditionContainerHealthy, t)
 	apitest.CheckConditionSucceeded(r.duck(), RevisionConditionResourcesAvailable, t)
 }
+
+func TestGetContainerConcurrency(t *testing.T) {
+	cases := []struct {
+		name   string
+		status RevisionSpec
+		want   int64
+	}{{
+		name:   "empty revisionSpec should return default value",
+		status: RevisionSpec{},
+		want:   0,
+	}, {
+		name: "get containerConcurrency by passing value",
+		status: RevisionSpec{
+			RevisionSpec: v1beta1.RevisionSpec{
+				ContainerConcurrency: ptr.Int64(10),
+			},
+		},
+		want: 10,
+	}}
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			if want, got := tc.want, tc.status.GetContainerConcurrency(); want != got {
+				t.Errorf("got: %v want: %v", got, want)
```

Comment:
We usually do

```suggestion
				t.Errorf("GetContainerConcurrency() = %v, want %v", got, want)
```

style assertions.

Reply:
I feel it deviates from other testcase